pkg/cmd/issue/develop/develop.go

```diff
@@ -106,7 +106,7 @@ func NewCmdDevelop(f *cmdutil.Factory, runF func(*DevelopOptions) error) *cobra.
 
 	fl := cmd.Flags()
 	fl.StringVar(&opts.BranchRepo, "branch-repo", "", "Name or URL of the repository where you want to create your new branch")
-	fl.StringVarP(&opts.BaseBranch, "base", "b", "", "Name of the base branch you want to make your new branch from")
+	fl.StringVarP(&opts.BaseBranch, "base", "b", "", "Name of the remote base branch you want to make your new branch from")
```

Comment:
Minor grammar nit
```suggestion
	fl.StringVarP(&opts.BaseBranch, "base", "b", "", "Name of the remote branch you want to make your new branch from")
```

Reply:
@andyfeller Thanks for your review. I've fixed it.